crates/hir-expand/Cargo.toml

```diff
@@ -33,6 +33,7 @@ mbe.workspace = true
 limit.workspace = true
 span.workspace = true
 parser.workspace = true
+ra-ap-rustc_lexer.workspace = true
```

Comment:
Let's re-export the unescape stuff from the parser or syntax crate (reason being that the `ra-ap-rustc` crates are special and require extra `extern crate` prelude annotations, a re-export saves us from that here